goverge/coverage.py

```diff
@@ -205,7 +205,8 @@ def get_package_deps(project_package, test_path, tag):
     package_deps = [
         package.replace("]", "").replace("[", "").replace("'", "")
         for package in list(set(output))
-        if project_package.split("\n")[0] in package]
+        if project_package.split("\n")[0] in package and
+        project_package.split("\n")[0]+"/vendor" not in package]
```

Comment:
this feels like list comprehension abuse to me, maybe something slightly different and more clear?

I've tried rewriting it a few times but am not sure enough what's going on to name variables. Performance here isn't really an issue I wouldn't think so maybe something like:

```
something = project_package.split("\n")[0]
package_deps = []
for package in list(set(output)):
    p = package.replace("]", "").replace("[", "").replace("'", "")
    if something in p and (something + "/vendor") not in p:
        package_deps.append(p)
```

is just a lot easier to read